Pipfile

```diff
@@ -19,3 +19,4 @@ paramiko = "*"
 pytz = "*"
 requests = {extras = ["security"]}
 lochness = {path = ".", editable = true}
+bpanonymize = "*"
```

Comment:
The word is `dp...`, why is it `bp...`?

Reply:
Thanks for spotting this. It's typo. I'll fix all of your comments once your review is done.